src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -191,7 +191,7 @@ private BoundExpression CreateUserDefinedConversion(SyntaxNode syntax, BoundExpr
                 conversion: conversion.UserDefinedFromConversion,
                 isCast: false,
                 conversionGroup,
-                wasCompilerGenerated: true,
+                wasCompilerGenerated: false,
```

Comment:
This change makes the results of this conversion visible from the semantic model. See the final test in `NullablePublicAPITests.MultipleConversions`, where I'm testing that the type is `B` and the converted type is `A`. Before this change, they were both reported as `B` because the conversion was considered compiler-generated and therefore ignored by the semantic model.